Intersect (Core)/Config/ProcessingOptions.cs

```diff
@@ -103,6 +103,11 @@ public partial class ProcessingOptions
         /// </summary>
         public int CommonEventAutorunStartInterval { get; set; } = 500;
 
+        /// <summary>
+        /// How often should the server check for stale cooldowns on players to remove them from their respective collections.
+        /// </summary>
+        public int StaleCooldownRemovalTimer { get; set; } = 60000;
```

Comment:
Can you add code to the Validate method to make sure this value is at least 100 so this doesn't run every tick?

Reply:
Would it be fine to have it throw an InvalidOperationException like the rest?
If so, what would a good error description be?

Reply:
Settled on "StaleCooldownRemovalTimer is too low, value should be at least 100ms to avoid running this check every server tick." on Discord.